internal_ws/ykcompile/src/lib.rs

```diff
@@ -139,7 +139,10 @@ pub struct CompiledTrace {
 }
 
 impl CompiledTrace {
-    /// Execute the trace by calling (not jumping to) the first instruction's address.
+    /// Execute the trace by calling (not jumping to) the first instruction's address. Returns a
```

Comment:
Can this return `null` if there is no guard failure?

In the future the only way out of the trace is via a guard failure, but I don't think that's the case now?

Reply:
You're right. Fixed in 2173f3a.